### DISCUSSION TITLE
Error Installing Foundry

### DISCUSSION THREAD

**Discussion by FFFarhan** [Cyfrin/foundry-full-course-cu#2486] [Q&A]
i get a permission denied error whe i try to install Foundry.
Using curl i get :- 
```
curl -L https://foundry.paradigm.xyz | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   167  100   167    0     0    299      0 --:--:-- --:--:-- --:--:--   299
100  2189  100  2189    0     0   1390      0  0:00:01  0:00:01 --:--:--  2853
Installing foundryup...
mkdir: cannot create directory ‘/home/farhan/.foundry’: Permission denied

```

and using wget i get :- 
```
wget -O foundryup-install.sh https://foundry.paradigm.xyz
foundryup-install.sh: Permission denied
```

Im on Linux Mint(ubuntu)
Please help me out here!!

**Comment by EngrPips** [comment]
Hello @FFFarhan, Please read this `stack-overflow` post and see if it will help you https://stackoverflow.com/questions/24005974/cannot-use-mkdir-in-home-directory-permission-denied-linux-lubuntu

**Reply by FFFarhan** [reply]
Ive already seen this. It doesnt help here :(

**Reply by EngrPips** [reply]
This error `mkdir: cannot create directory ‘/home/farhan/.foundry’: Permission denied\` is explicit, though I think the current user is just missing write permission to the directory where Foundry wants to write a file. Have you seen this post https://forum.jetbackup.com/viewtopic.php?t=466, too?

**Reply by FFFarhan** [reply]
> This error `mkdir: cannot create directory ‘/home/farhan/.foundry’: Permission denied\` is explicit, though I think the current user is just missing write permission to the directory where Foundry wants to write a file. Have you seen this post https://forum.jetbackup.com/viewtopic.php?t=466, too?

I see,
i checked it out and i tried manually making a directory, I apparently cant even make it manually

**Reply by EngrPips** [reply]
I am unsure how to resolve this. @PatrickAlphaC, we could use some help, sir.
